src/lexedata/importer/fromexcel.py

```diff
@@ -218,10 +215,10 @@ def properties_from_row(
 
         # fetch cell comment
         comment = get_cell_comment(row[0])
-        properties[c_r_comment] = comment
+        properties["Comment"] = comment
```

Comment:
Nobody guarantees you that the column is called 'Comment'. Why would you do this?
Note that the c_r_comment variable will contain the string 'Comment' if and only if that column is actually named 'Comment', that is the magic of `self.db.dataset[RowObject.__table__, "comment"].name`, which maps the term (`comment`) back to the column name (`Comment`, or `Comments`, or `Notiz`, or whatever).

Reply:
I think, I was just confused as the style of cellparser changed, so I adapted it in fromexcel.py to be consistent.